src/dbos-runtime/runtime.ts

```diff
@@ -40,7 +41,10 @@ export class DBOSRuntime {
       this.dbosExec = new DBOSExecutor(this.dbosConfig);
       this.dbosExec.logger.debug(`Loading classes from entrypoints ${JSON.stringify(this.runtimeConfig.entrypoints)}`);
       const classes = await DBOSRuntime.loadClasses(this.runtimeConfig.entrypoints);
-      await this.dbosExec.init(...classes);
+      for (const cls of getAllRegisteredClasses()) {
+        if (!classes.includes(cls)) classes.push(cls);
+      }
```

Comment:
So why would we ever need line 43 if we can just do this?

Reply:
If you don't do a load of something, then the decorators won't run, and nothing will get registered.  If you bring in a source file and that brings in all its dependencies, then you can call getAllRegisteredClasses and see what the decorators decorated.

Reply:
I think we could ignore the return value from line 43, and do less work in that call maybe, but you still would import the entry points files, then get the registered classes.